fvm/src/gas/price_list.rs

```diff
@@ -416,9 +526,74 @@ impl PriceList {
     pub fn on_verify_consensus_fault(&self) -> GasCharge<'static> {
         GasCharge::new("OnVerifyConsensusFault", self.verify_consensus_fault, 0)
     }
+
+    /// Returns the gas required for the specified amount of fuel.
+    #[inline]
+    pub fn on_consume_fuel(&self, fuel: u64) -> GasCharge<'static> {
+        GasCharge::new("OnConsumeFuel", self.gas_per_fuel * fuel as i64, 0)
```

Comment:
Can we call this "execution units" (like in FIP-0032) (maybe abbreviate to ExecUnits). Fuel is a wasmtime-specific term which won't apply to other engines.

We may want to remove the term `fuel` from the interface of our components too -- we should only use this term privately in the context of wasmtime.